PlasmaEntity.js

```diff
@@ -4,7 +4,7 @@ const uuidv4 = require('uuid/v4');
 class PlasmaEntity {
 
     constructor(){
-        this._uuid = null;
+        this._uuid = uuidv4();
```

Comment:
I made the uuid generate on instantiation to make the .initialize() function redundant. This uuid will get overwritten if the entity is populated from the database anyways. I did this so that when we create a new entity instantiation its cleaner. If you are worried about doing an unnecessary uuid generation when loading data from the database we can implement a different alternative to having to call .initialize() after creating a new instance of an entity